app/serializers/articles_index_serializer.rb

```diff
@@ -1,11 +1,14 @@
 class ArticlesIndexSerializer < ActiveModel::Serializer
-  attributes :id, :title, :teaser, :date, :category, :language, :image, :rating, :author, :premium, :published,
-             :comments
+  attributes :id, :title, :teaser, :date, :category, :status, :language, :image, :rating, :author, :premium, :comments
 
   def date
```

Comment:
Didn't see `:status` being added to the BackyardArticles index serializer - did I miss it?

Reply:
Nice spot! we forgot it. Is fixed now

Reply:
🚀